app/src/organisms/Desktop/Devices/ProtocolRun/ProtocolRunCamera/ImageGalleryContainer/GalleryItemCard.tsx

```diff
@@ -56,11 +140,21 @@ export function GalleryItemCard({
         onClick={onClick}
         role="button"
       >
-        <img
-          className={styles.gallery_img}
-          src={imagePath}
-          alt="camera-photo"
-        />
+        {imagePath != null && !isLoading ? (
+          <img
+            className={styles.gallery_img}
+            src={imagePath}
+            alt="camera-photo"
+          />
+        ) : (
+          <div className={styles.gallery_img_placeholder}>
+            <Skeleton
+              width="100%"
+              height="100%"
+              backgroundSize="47rem"
+            ></Skeleton>
+          </div>
+        )}
         <div className={styles.gallery_img_overlay}>
```

Comment:
```suggestion
        {imagePath != null && !isLoading && <div className={styles.gallery_img_overlay}> ...
```
If we're showing the skeleton, let's hide that "view image" hover state.